variables.tf

```diff
@@ -22,12 +22,17 @@ variable "acl" {
   default     = "private"
 }
 
+variable "bucket_policy" {
+  description = "The S3 bucket policy to set. If empty, no policy will be set"
+  default     = ""
+}
+
```

Comment:
Can we put a commented-out example of what it would look like if you put a real S3 policy in there?

Reply:
Do you think it would fit better in the actual example (still commented out)? I'm not sure that users of the module actually look into these files, however, placing it in the example could result in it being replicated in every invocation of the module.

That being said, I'm not sure what example I would use. I don't have experience with S3 bucket policies and what the typical use-cases are so I would have to choose something simple from the AWS docs (if they have examples). Do you think that's okay? Alternatively, do you have any suggestions?

Reply:
The formbuilder & analytical platform folks are the ones who wanted this feature. Let's just grab whatever they come up with and use that, commented out in the example.

Personally, I wouldn't worry about multiple copies of comments ending up in the environments repo. It won't do any harm, even if people don't delete the comments, and if we really want to, we can always force them to do it. We're the ones who have to approve the PRs, after all!